.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
I have a package-lock locally I plan to commit soon. Let's remove `.npmrc` for now.